Estore.UITests/Features/CreateUsersTests.feature

```diff
@@ -0,0 +1,136 @@
+﻿Feature: CreateUsersTests
+
+As a Admin
+I would like to create new customer via UI
+
+#US_49_25a
+@AdminLoggedIn
+Scenario: US_49_25a_Admin abserve the layout and elements on the Create user modal window
+	Given Admin click on the Users button
+	And User page is open
+	When Admin click on the Add User button
+	Then Create user modal window should be open
+	And Create user modal window contains fields: 'First name, Last name, Email, Password, Password'
+	And Register button should be disabled
+	And Help message under 'Password' field should be 'Choose a strong password'
+	And Help message under 'Repeat password' field should be 'Repeat the password'
+
+#US_49_25b
+@AdminLoggedIn
+Scenario: US_49_25b_Admin closes Create user modal window
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin click on the Cancel button
+	Then Create user modal window should be close
+
+#US_49_26
+@AdminLoggedIn
+Scenario: US_49_26_Admin adds a new customer
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills modal window and registers new customer
+	Then Create user modal window should be close
+	Then New customer appeared in the users list
+	And Info message 'User was added' is presented
+
+#US_49_27
+@AdminLoggedIn
+Scenario: US_49_27_Admin leaves First name field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'First name' input field: '' 
+	Then Help message under 'First name' field should be 'First name is required!'
+
+#US_49_28
+@AdminLoggedIn
+Scenario: US_49_28_Admin leaves Last name field empty
+	Given Admin click on the Users button
+	And User page is open
+	And Admin click on the Add User button
+	When Create user modal window is open
+	And Admin fills 'Last name' input field: '' 
+	Then Help message under 'Last name' field should be 'Last name is required!'
```

Comment:
May be it is possible to combine this scenarios to scenario outline with examples

Reply:
and some tests below

Reply:
fixed